features/step_definitions/user_steps.rb

```diff
@@ -331,6 +331,19 @@ def get_series_name(age, classname, name)
   step(%{the email should contain "The username for your .* has been changed to"})
 end
 
+When "I change my email to {string}" do |new_email|
+  step %{I follow "My Preferences"}
+  step %{I follow "Change Email"}
+  fill_in("New email", with: new_email)
+  fill_in("Enter new email again", with: new_email)
+  fill_in("Password", with: "password")
+  click_button("Confirm New Email")
+  click_button("Yes, Change Email")
```

Comment:
```suggestion
step %{I request to change my email to "#{new_email}"}
```

and you could move it to `profile_steps.rb`